routers/api/v1/org/org.go

```diff
@@ -315,6 +316,52 @@ func Get(ctx *context.APIContext) {
 	ctx.JSON(http.StatusOK, org)
 }
 
+func Rename(ctx *context.APIContext) {
+	// swagger:operation POST /orgs/{org}/rename organization renameOrg
+	// ---
+	// summary: Rename an organization
+	// produces:
+	// - application/json
+	// parameters:
+	// - name: org
+	//   in: path
+	//   description: existing org name
+	//   type: string
+	//   required: true
+	// - name: body
+	//   in: body
+	//   required: true
+	//   schema:
+	//     "$ref": "#/definitions/RenameOrgOption"
+	// responses:
+	//   "204":
+	//     "$ref": "#/responses/empty"
+	//   "403":
+	//     "$ref": "#/responses/forbidden"
+	//   "422":
+	//     "$ref": "#/responses/validationError"
+
+	org := ctx.Org.Organization
+	form := web.GetForm(ctx).(*api.RenameOrgOption)
+
+	oldName := org.AsUser().Name
+
+	if err := user_service.RenameUser(ctx, org.AsUser(), form.NewName); err != nil {
+		if user_model.IsErrUserAlreadyExist(err) {
+			ctx.Error(http.StatusUnprocessableEntity, "RenameOrg", ctx.Tr("form.username_been_taken"))
```

Comment:
IIRC we never use `ctx.Tr` for `ctx.Error(http.StatusUnprocessableEntity,`

Reply:
@wxiaoguang , the rename user api uses it (see screenshot attached).

![image](https://github.com/user-attachments/assets/6825f666-5ab7-4c0d-afcb-ff3e90908a75)

Reply:
It's caused by copy-paste .... I do not think it's right to use `ctx.Tr` here & there.